Frends.SMTP.SendEmail/Frends.SMTP.SendEmail/Definitions.cs

```diff
@@ -3,173 +3,207 @@
 
 namespace Frends.SMTP.SendEmail
 {
+    /// <summary>
+    /// Input parameters.
+    /// </summary>
     public class Input
     {
         /// <summary>
         /// Recipient addresses separated by ',' or ';'
         /// </summary>
-        [DefaultValue("\"jane.doe@somedomain.com\"")]
+        /// <example>jane.doe@somedomain.com;john.doe@somedomain.com</example>
         public string To { get; set; }
 
         /// <summary>
         /// Cc recipient addresses separated by ',' or ';'
         /// </summary>
-        [DefaultValue("\"jane.doe@somedomain.com\"")]
+        /// <example>jane.doe@somedomain.com;john.doe@somedomain.com</example>
         public string Cc { get; set; }
 
         /// <summary>
         /// Bcc recipient addresses separated by ',' or ';'
         /// </summary>
-        [DefaultValue("\"jane.doe@somedomain.com\"")]
+        /// <example>jane.doe@somedomain.com;john.doe@somedomain.com</example>
         public string Bcc { get; set; }
 
         /// <summary>
         /// Sender address.
         /// </summary>
-        [DefaultValue("\"john.doe@somedomain.com\"")]
+        /// <example>foo.bar@somedomain.com</example>
         public string From { get; set; }
 
         /// <summary>
         /// Name of the sender.
         /// </summary>
-        [DefaultValue("\"\"")]
+        /// <example>Foo Bar</example>
         public string SenderName { get; set; }
 
         /// <summary>
         /// Email message's subject.
         /// </summary>
-        [DefaultValue("\"Hello Jane\"")]
+        /// <example>Hello</example>
         public string Subject { get; set; }
 
         /// <summary>
         /// Body of the message.
         /// </summary>
-        [DefaultValue("\"You've got mail!\"")]
+        /// <example>You've got mail</example>
         public string Message { get; set; }
 
         /// <summary>
         /// Set this true if the message is HTML.
         /// </summary>
-        [DefaultValue("false")]
+        /// <example>false</example>
+        [DefaultValue(false)]
         public bool IsMessageHtml { get; set; }
 
         /// <summary>
         /// Encoding of message body and subject. Use following table's name column for other options. https://msdn.microsoft.com/en-us/library/system.text.encoding(v=vs.110).aspx#Anchor_5 
         /// </summary>
-        [DefaultValue("\"utf-8\"")]
+        /// <example>utf-8</example>
+        [DefaultValue("utf-8")]
         public string MessageEncoding { get; set; }
-
     }
 
+    /// <summary>
+    /// Options parameters.
+    /// </summary>
     public class Options
```

Comment:
You could also refactor the code by moving all of these classes into their own files.